src/main/java/microsoft/exchange/webservices/data/Base64EncoderStream.java

```diff
@@ -23,69 +23,15 @@
 
 package microsoft.exchange.webservices.data;
 
+import org.apache.commons.codec.binary.Base64;
+import org.apache.commons.codec.binary.StringUtils;
+
 import java.util.Vector;
 
 /**
  * The Class Base64EncoderStream.
  */
 class Base64EncoderStream {
```

Comment:
It could have, was going for the least churn.  If you'd like, I'll inline the calls at all the call sites.